ArmTemplates/Application-Insights/slack-alerts-logic-app.json

```diff
@@ -0,0 +1,135 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "instanceBaseName": {
+        "type": "string",
+        "metadata": {
+          "description": "The base name for resources."
+        }
+      },
+      "slackChannel": {
+        "type": "string",
+        "defaultValue": "#general",
```

Comment:
I think this should be a required rather than optional param.  The channel the alerts are posted to should be a deliberate choice.

Reply:
Would be useful to state in the docs that the channel name should be prefixed with #